apps/web/src/services/alert-manager/v2/components/AlertsManagementTable.vue

```diff
@@ -190,13 +192,67 @@ const filterState = reactive({
     ])),
 });
 
-const alertListApiQueryHelper = new ApiQueryHelper().setSort('created_at', true)
-    .setPage(1, 15);
+const alertListApiQueryHelper = new ApiQueryHelper().setSort('created_at', true);
 
 const filterQueryHelper = new QueryHelper();
 const queryTagHelper = useQueryTags({ keyItemSets: ALERT_MANAGEMENT_TABLE_HANDLER.keyItemSets });
 const { queryTags } = queryTagHelper;
 
+const { alertAPI } = useAlertApi();
+const { key: alertListQueryKey, params: alertListQueryParams } = useServiceQueryKey('alert-manager', 'alert', 'list', {
+    params: computed(() => {
+        alertListApiQueryHelper.setPage(filterState.pageStart, filterState.pageLimit);
+        filterQueryHelper.setFilters([]);
+        if (storeState.selectedUrgency !== 'ALL') {
+            filterQueryHelper.addFilter({ k: 'urgency', v: storeState.selectedUrgency, o: '=' });
+        }
+        if (storeState.selectedStatus === ALERT_STATUS_FILTERS.OPEN) {
+            filterQueryHelper.addFilter({ k: 'status', v: [ALERT_STATUS_FILTERS.TRIGGERED, ALERT_STATUS_FILTERS.ACKNOWLEDGED], o: '=' });
+        } else if (storeState.selectedStatus !== 'ALL') {
+            filterQueryHelper.addFilter({ k: 'status', v: storeState.selectedStatus, o: '=' });
+        }
+        if (storeState.selectedLabels?.length > 0) {
+            filterQueryHelper.addFilter({ k: 'labels', v: storeState.selectedLabels.map((i) => i.name), o: '=' });
+        }
+
+        if (storeState.period.start && storeState.period.end && (storeState.period.start === storeState.period.end)) {
+            filterQueryHelper.addFilter({ k: 'created_at', v: storeState.period.start, o: '>=' });
+        } else {
+            if (storeState.period.start) {
+                filterQueryHelper.addFilter({ k: 'created_at', v: storeState.period.start, o: '>=' });
+            }
+            if (storeState.period.end) {
+                filterQueryHelper.addFilter({ k: 'created_at', v: storeState.period.end, o: '<=' });
+            }
+        }
+
+        if (state.isServicePage) {
+            filterQueryHelper.addFilter({ k: 'service_id', v: storeState.serviceId, o: '=' });
+        } else if (storeState.selectedServiceId) {
+            filterQueryHelper.addFilter({ k: 'service_id', v: storeState.selectedServiceId, o: '=' });
+        }
+
+        alertListApiQueryHelper.setFilters([
+            ...queryTagHelper.filters.value,
+            ...filterQueryHelper.filters,
+        ]);
+
+        return {
+            query: alertListApiQueryHelper.data,
+        };
+    }),
+});
+const { data: alertListData, isLoading: alertListLoading, refetch: refetchAlertList } = useQuery({
+    queryKey: alertListQueryKey.value,
+    queryFn: async () => {
+        const res = await alertAPI.list(alertListQueryParams.value);
+        return {
+            results: res.results ?? [],
+            totalCount: res.total_count ?? 0,
+        };
+    },
+    enabled: false,
+});
```

Comment:
- Suggestion: Rather than adding a default value through queryFn, why don't we do defense where we write the value? I think processing the data received from the server and putting it in the cache can undermine the consistency of the cache.
---
queryFn을 통해 default값을 넣어주는 것보다는, 값을 쓰는 곳에서 방어처리를 해주는게 어떨까요? 서버로부터 받은 데이터를 가공해서 캐시에 넣는 것은 캐시의 정합성을 해칠 수 있을 것 같습니다.

```typescript
const { data: alertListData, isLoading: alertListLoading, refetch: refetchAlertList } = useScopedQuery({
    queryKey: alertListQueryKey,
    queryFn: async () =>  alertAPI.list(alertListQueryParams.value),
    enabled: false,
    staleTime: 1000 * 60 * 5, // example
});
```

Reply:
- Please apply staleTime (also GCTime if needed)! Please use `useScopedQuery`!
--- 
staleTime(필요하다면 GCTime도)을 적용해주세요! `useScopedQuery`를 사용해주세요!